### PR TITLE
[terra-functional-testing] move code from lib to src

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary

Move code from lib to src in prep of adding the visual-regression-service that needs to be transpiled.

[CONTRIBUTORS.md]: ../blob/main/CONTRIBUTORS.md
[License]: ../blob/main/LICENSE

### CODE REVIEW

**Review by mjhenkes — APPROVED** [review]

File: packages/terra-functional-testing/CHANGELOG.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 ## Unreleased
 
-* Initial stable release
+* Initial stable release!
```
[the remark this answers is not in the document]
Comment (tbiethman) [reply packages/terra-functional-testing/CHANGELOG.md:5]: 🎉

**Review by tbiethman — APPROVED** [review]

File: packages/terra-functional-testing/bin/test-runner.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env node
 
-require('../lib/cli').run();
+require('../src/cli').run();
```
Comment (StephenEsser) [inline packages/terra-functional-testing/bin/test-runner.js:3]: Is this going to switch back to lib in the next PR? I think we'll run into issues using /src/ when this is consumed as a released package

Comment (emilyrohrbough) [reply packages/terra-functional-testing/bin/test-runner.js:3]: Yeah, I will update when we pull babel in.

**Pull request merged**